website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,235 @@
+---
+name: 'ACL Replication for Multiple Datacenters'
+content_length: 15
+id: acl-replication
+layout: content_layout
+products_used:
+  - Consul
+description: You can configure tokens, policies, and roles to work across multiple datacenters.
+---
+
+You can configure tokens, policies, and roles to work across multiple
+datacenters. This provides central configuration of security policies in the
+primary datacenter and failover for the secondary datacenters. If the primary
+datacenter goes offline, the secondary datacenters will still have the security
+policies in place. 
+
+~> Note, each datacenter will need to have ACLs enabled, the process is outlined in the [Securing Consul with ACLs
```

Comment:
We should probably label these as prerequisites and not a note (note formatting is fine, but it would be good to have them as a bulleted list if that's possible inside the box.

Reply:
I'll update the formatting.